src/test/java/wolox/training/repositoriesTest/UserRepositoryTest.java

```diff
@@ -0,0 +1,55 @@
+package wolox.training.repositoriesTest;
```

Comment:
This shouldn't be here

Reply:
Please review again, this branch wasn't update with main.